efficientdet/dataloader.py

```diff
@@ -394,7 +394,10 @@ def __call__(self, params):
 
     # Prefetch data from files.
     def _prefetch_dataset(filename):
-      dataset = tf.data.TFRecordDataset(filename).prefetch(1)
+      if params.get('dataset_type', None) == 'sstable':
```

Comment:
'sstable'?

Reply:
Yeah, it is an internal experimental feature.